src/main/resources/application.properties

```
@@ -1 +1,2 @@
 spring.application.name=ecommerce
+logging.file.name=arquivo.log
```

podemos alterar o nome desse arquivo para inglês? só para manter o padrão do projeto...

sugestão:

```suggestion
logging.file.name=logfile.log
```